src/app/games/coerceo/coerceo.component.ts

```diff
@@ -56,6 +57,10 @@ export class CoerceoComponent extends TriangularGameComponent<CoerceoRules,
         this.scores = MGPOptional.of(PlayerNumberMap.of(0, 0));
     }
 
+    public override getScoreName(): string {
+        return ScoreName.CAPTURES();
```

Comment:
erh, "1 captures"

Reply:
wow good point !  not so small refactor pour en tenir compte